storage/remote/codec_test.go

```diff
@@ -125,15 +125,10 @@ func TestValidateLabelsAndMetricName(t *testing.T) {
 		t.Run(test.description, func(t *testing.T) {
 			err := validateLabelsAndMetricName(test.input)
 			if err == nil {
-				if !test.shouldPass {
-					t.Fatalf("Test should fail, but passed instead.")
-				}
+				testutil.Assert(t, test.shouldPass, "Test should fail, but passed instead.")
```

Comment:
This would be way more cleaner and consistent if we would change it a bit:

```
if test.shouldPass {
   testutil.Ok(..
else {
   testutil.NotOk(..)
  testutil.Equals(t, test.expectedErr, err.Error())
}
```

Also Can we remove `shouldPass` field and just check if `expectedErr` is nil? (: What do you think?

Reply:
I second these comments, the flow is weird right now, and `shouldPass` is definitely duplicated information.

Reply:
Done!